docs/docs/coverage/language/dotnet.md

```diff
@@ -11,11 +11,13 @@ The following scanners are supported.
 
 The following table provides an outline of the features Trivy offers.
 
-| Package manager | File               | Transitive dependencies | Dev dependencies | [Dependency graph][dependency-graph] | Position |
-|:---------------:|--------------------|:-----------------------:|:----------------:|:------------------------------------:|:--------:|
-|    .Net Core    | *.deps.json        |            ✓            |     Excluded     |                  -                   |    ✓     |
-|      NuGet      | packages.config    |            ✓            |     Excluded     |                  -                   |    -     |
-|      NuGet      | packages.lock.json |            ✓            |     Included     |                  ✓                   |    ✓     |
+| Package manager | File                         | Transitive dependencies | Dev dependencies | [Dependency graph][dependency-graph] | Position |
+|:---------------:|------------------------------|:-----------------------:|:----------------:|:------------------------------------:|:--------:|
+|    .Net Core    | *.deps.json                  |            ✓            |     Excluded     |                  -                   |    ✓     |
+|      NuGet      | packages.config              |            ✓            |     Excluded     |                  -                   |    -     |
+|      NuGet      | packages.props               |            -            |     Excluded     |                  -                   |    -     |
+|      NuGet      | Directory.packages.props     |            -            |     Excluded     |                  -                   |    -     |
```

Comment:
```suggestion
|      NuGet      | *packages.props               |            -            |     Excluded     |                  -                   |    -     |
```

1 line for packages.props will be enough.

But can you add new section (as for `*.deps.json`, `packages.config`, etc.).
In this section you can write about `Directory.Packages.props`, that  `Packages.props` is outdated, etc.